packages/babel-helper-define-map/src/index.js

```diff
@@ -65,8 +65,8 @@ export function push(mutatorMap: Object, node: Object, kind: string, file, scope
     value = nameFunction({ id: key, node: value, scope });
 
     // Class methods don't have their name bound in the funciton body.
-    if (t.isClassMethod(node)) {
-      value.id[t.NOT_LOCAL_BINDING] = true
+    if (t.isClassMethod(node) && value.id) {
```

Comment:
Ok although it looks like the id in this case is null since we specifically pass `null` here https://github.com/babel/babel/pull/3289/files#diff-9f4f530c18baf47c12f8f619ca3c69fbR55